Readme.md

```diff
@@ -45,7 +45,8 @@ Unlike regular objects, __keys will not be stringified__. For example numbers an
 - `keys() : Array<*>` returns an array with all the registered keys
 - `values() : Array<*>` returns an array with all the values
 - `entries() : Array<[*,*]>` returns an array with [key,value] pairs
-- `count() : Number` returns the amount of key-value pairs
+- `size : Number` returns the amount of key-value pairs
+- `count() : Number` alias for `size`

```

Comment:
It's... not exactly an alias because one is a property and the other a method. I say, in this case, just copy the same description. I think we could deprecate them by adding a `(deprecated)` at the end of the description and delete them in the future (same for `remove()`)